docs/user-guide/configuring.md

```diff
@@ -15,7 +15,7 @@ All of these options give you fine-grained control over how ESLint treats your c
 
 ## Specifying Parser Options
 
-ESLint allows you to specify the JavaScript language options you want to support. By default, ESLint supports only ECMAScript 5 syntax. You can override that setting to enable support for ECMAScript 6 and 7 as well as [JSX](http://facebook.github.io/jsx/) by using parser options.
+ESLint allows you to specify the JavaScript language options you want to support. By default, ESLint supports only ECMAScript 5 syntax. You can override that setting to enable support for ECMAScript 6/7/8 as well as [JSX](http://facebook.github.io/jsx/) by using parser options.
```

Comment:
I'll go with @nzakas' suggestion. We definitely don't need to list supported ecmaVersion values twice.